src/tests/tests/utils.js

```diff
@@ -65,6 +65,17 @@
     assert.ok(!badger.isPrivacyBadgerEnabled("example.com"), "disabled for site");
     badger.enablePrivacyBadgerForOrigin("example.com");
     assert.ok(badger.isPrivacyBadgerEnabled("example.com"), "enabled for site");
+
+    // whitelist wildcard cases
+    badger.disablePrivacyBadgerForOrigin('*.mail.example.com');
+    assert.ok(badger.isPrivacyBadgerEnabled('www.example.com'), "enabled for site");
```

Comment:
Would be nice to provide more descriptive messages. What are we asserting with each statement?